backend/src/plugins/Utility/functions/getServerInfoEmbed.ts

```diff
@@ -205,9 +205,14 @@ export async function getServerInfoEmbed(
         [GuildPremiumTier.Tier3]: 60,
       }[restGuild.premiumTier] ?? 0;
 
+    const availableEmojis = restGuild.emojis.cache.filter((e) => e.available);
     otherStats.push(
-      `Emojis: **${restGuild.emojis.cache.size}** / ${maxEmojis * 2}${
+      `Emojis: **${availableEmojis.size}** / ${maxEmojis * 2}${
         roleLockedEmojis ? ` (__${roleLockedEmojis} role-locked__)` : ""
+      }${
+        availableEmojis.size < restGuild.emojis.cache.size
+          ? ` (__${restGuild.emojis.cache.size - availableEmojis.size} unavailable__)`
```

Comment:
Maybe let's add a + here to explicitly say that the total emoji count does not include these
```suggestion
          ? ` (__+${restGuild.emojis.cache.size - availableEmojis.size} unavailable__)`
```